TriblerGUI/widgets/trustpage/js/radial_view/radial_inspector.js

```diff
@@ -0,0 +1,146 @@
+if (typeof require !== "undefined") {
+    var config = require("../style_config.js");
+}
+
+/**
+ * The data inspector for the Radial Network
+ * @param d3element - the container HTML element wrapped in d3
+ * @param {Object} options - configuration overrides for the links
+ * @constructor
+ */
+function RadialInspector(d3element, options) {
+
+    var self = this,
```

Comment:
this is a keyword that will be assigned in every function, it is difficult to predict what it will refer to inside callbacks or methods. When I assign it to self, I can use self everywhere and be sure it does not change. This is common practice in js